src/SWP/UpdaterBundle/Model/UpdatePackage.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Updater Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\UpdaterBundle\Model;
+
+use Updater\Package\Package;
+
+/**
+ * Update package model class.
+ */
+class UpdatePackage extends Package
```

Comment:
needed to set some extra property (url) which is not used by the ahilles107/updater, but is needed for UpdaterBundle purposes